### PR TITLE
Feature/rdsssam 236 form display changes

### PR DESCRIPTION
**Pull request by dannyberzon**
Some screen shots of the relevant parts of the form:

screen shot 2018-02-23 at 12 26 06
screen shot 2018-02-23 at 12 25 57

### CODE REVIEW

File: willow/app/views/hyrax/rdss_cdms/_attribute_rows.html.erb [hunk 1]
```diff
@@ -1,18 +1,18 @@
 <%= presenter.attribute_to_html(:title) %>
 <%= presenter.attribute_to_html(:object_description) %>
-<%= presenter.attribute_to_html(:object_keywords) %>
-<%= presenter.attribute_to_html(:object_category) %>
 <%= presenter.attribute_to_html(:object_resource_type, render_as: :humanized_faceted) %>
 <%= presenter.attribute_to_html(:object_value, render_as: :humanized) %>
-<%= presenter.attribute_to_html(:object_organisation_roles, render_as: :object_organisation_roles, label: I18n.t('willow.fields.object_organisation_role')) %>
 <%= presenter.attribute_to_html(:object_people, render_as: :object_people, label:I18n.t('willow.fields.object_people')) %>
 <%= presenter.attribute_to_html(:object_dates, render_as: :object_dates, label: I18n.t('willow.fields.object_dates')) %><%# the `:render_as` parameter tells the presenter to use the object_dates_attribute_renderer %>
```
Comment (blackrat) [inline willow/app/views/hyrax/rdss_cdms/_attribute_rows.html.erb]: I thought dates was going higher.

File: willow/config/locales/rdss_cdm.en.yml [hunk 2]
```diff
@@ -47,6 +48,7 @@ en:
         # would return a hash
         licence: Licensing and distribution information governing access to the dataset.
         rights_holder: The person or organisation who holds the rights for the work
+        jisc_id: Enter an Organisation Identifier if applicable
```
Comment (blackrat) [inline willow/config/locales/rdss_cdm.en.yml]: downcase Identifier

File: willow/app/views/hyrax/rdss_cdms/_attribute_rows.html.erb (hunk 1 above)
Comment (dannyberzon) [reply willow/app/views/hyrax/rdss_cdms/_attribute_rows.html.erb]: Yeah good point, I'll fix

Comment (dannyberzon) [reply willow/app/views/hyrax/rdss_cdms/_attribute_rows.html.erb]: Actually, my bad. In the notes in the ACs it says:

"The object dates field should appear after people in the form"

So the current positioning is correct

**Pull request merged**